### DISCUSSION TITLE
Replace xN with names

### DISCUSSION THREAD

**Discussion by antonmedv** [community/community#18714] [Other Feature Feedback, Questions, & Ideas]
Hi!

Here is my suggestion: replace badges with numbers (x3, x4) with separate badges and names:

- Pull Shark x1 — Pull Baby Shark
- Pull Shark x2 — Pull Papa Shark
- …
- 

Or something like this. Each with separate badge drawings.

**Comment by thomscoder** [comment]
Hi @antonmedv 
Seems like your request is to introduce a sort of tier system (am I right ?)
This is already implemented and you can find the thresolds here: https://github.com/Schweinepriester/github-profile-achievements
